lib/site_prism/exceptions.rb

```diff
@@ -6,7 +6,8 @@ class NoUrlMatcherForPage < StandardError; end
 
   class InvalidUrlMatcher < StandardError
     def message
-      'Could not automatically match your URL. Templated port numbers are unsupported.'
+      "Could not automatically match your URL. \
+Templated port numbers are unsupported."
```

Comment:
this needs indenting once from the previous line

Reply:
in this case, you'll have extra spaces in the message.
Now,` "Could not automatically match your URL. Templated port numbers are unsupported."`
if ident `"Could not automatically match your URL.          Templated port numbers are unsupported."`